### DISCUSSION TITLE
Enhancements for MQTT-Subscription Tutorial

### DISCUSSION THREAD

**Discussion by wjohn007** [tasmota/Berry_playground#34] [Tutorials and Code Blocks]
Please find some enhancements for MQTT-Subscription

https://github.com/wjohn007/Berry_playground/tree/main/tutorial-code-blocks/mqtt

John

**Comment by dkebler** [comment]
Thanks for your contribution.   I'll give you any input and then you can make a pr.

**Comment by wjohn007** [comment]
PR Done
